stdlib/Printf/src/Printf.jl

```diff
@@ -863,7 +863,7 @@ Use shorter of decimal or scientific 1.23 1.23e+07
 ```
 
 For a systematic specification of the format, see [here](https://www.cplusplus.com/reference/cstdio/printf/).
-See also [`@sprintf`](@ref).
+See also [`@sprintf`](@ref) to generate a `String`.
```

Comment:
```suggestion
See also [`@sprintf`](@ref) to generate a `String`, or [`IOBuffer`](@ref) to create a growable in-memory buffer that you can print to.
```

Reply:
Better to refer to `sprint` in that case IMO. But this is perhaps getting a bit excessive...